README.adoc

```diff
@@ -46,18 +163,35 @@ Synposis
 
 * Specify the desired credentials for the first connection to Rudder in *roles/ansible-deploy-rudder/vars/main.yml*. Here is an example of an account:
 
+----
+rudder_username: admin
+rudder_password_hash_type: bcrypt
+rudder_password_hash: $2b$12$uU.LawnF2lLlD4mAZviRKeNEb2Z58asv2d5QoEwl7N71Frrt0laJC
+rudder_user_role: administrator
+----
 
-        rudder_username: admin
-        rudder_password_hash_type: bcrypt
-        rudder_password_hash: $2b$12$uU.LawnF2lLlD4mAZviRKeNEb2Z58asv2d5QoEwl7N71Frrt0laJC
-        rudder_user_role: administrator
+* Then execute the `main_playbook.yml` using the following command:
 
+----
+ansible-playbook -i inventory main_playbook.yml
+----
 
+There's a few default configuration values defined in `tasks/configure.yml` file, which you can modify as well depending on your needs.
 
-* Then execute the `main_playbook.yml` using the following command:
+== Developer
 
+=== Environment setup
 
-        ansible-playbook -i inventory main_playbook.yml
+==== Prerequisites via apt
+Due to dependencies (for example *ansible* -> *paramiko* -> *pynacl* -> *libffi*):
 
+        sudo apt update
+        sudo apt install build-essential libssl-dev libffi-dev python-dev
 
-There's a few default configuration values defined in `tasks/configure.yml` file, which you can modify as well depending on your needs.
+==== Common environment setup
+        . Clone this repository: `git clone https://github.com/Normation/rudder-ansible.git`
+        . Change directory into the repository root dir: `cd rudder-ansible`
+        . Create a virtual environment: `python3 -m venv venv` (or for Python2: `virtualenv venv`. Note this requires you to install the virtualenv package: `pip install virtualenv`)
+        . Activate the virtual environment: `. venv/bin/activate`
+        . Install development requirements: `pip install -r test-requirements.txt`
+        . Exit the virtual environment when you are done: `deactivate`
```

Comment:
Explaining how to debug also would be great !

Reply:
This part will be modified when adding the debugging functionality.